client/src/App.js

```diff
@@ -20,6 +20,7 @@ import LikedVideosPage from './components/pages/LikedVideosPage/LikedVideosPage'
 import PlaylistsPage from './components/pages/PlaylistsPage/PlaylistsPage';
 import LoginPage from './components/pages/LoginPage/LoginPage';
 import SignupPage from './components/pages/SignupPage.jsx/SignupPage';
+import HistoryPage from './components/pages/HistoryPage/HistoryPage';
```

Comment:
Try importing it from one file like creating an index.js file it will look much cleaner